commit 5cbe23a00e1af14d5c23f799d4859191f21595af (refs/changes/14/2914/2)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2017-11-06T18:33:31+00:00 (1 year, 11 months ago)
    
    DAPI-346 sdc-designation tests are failing in nightly-1: tests 'allocate 1' and 'allocate 2'
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/test/integration.test.js b/test/integration.test.js
index ff084c2..edb597f 100644
--- a/test/integration.test.js
+++ b/test/integration.test.js
@@ -190,7 +190,7 @@ test('allocate 1', function (t) {
 			}
 		}, {
 			/* JSSTYLED */
-			step: 'Servers which meet image manifest platform requirements',
+			step: 'Servers which meet image manifest and package platform requirements',
 			remaining: [
 				'00000000-0000-0000-0000-00259094373c',
 				'00000000-0000-0000-0000-0025909437d4'
@@ -377,7 +377,7 @@ test('allocate 2', function (t) {
 			}
 		}, {
 			/* JSSTYLED */
-			step: 'Servers which meet image manifest platform requirements',
+			step: 'Servers which meet image manifest and package platform requirements',
 			remaining: [
 				'00000000-0000-0000-0000-00259094373c',
 				'00000000-0000-0000-0000-0025909437d4'
